perma_web/perma/urls.py

```diff
@@ -70,11 +70,16 @@
     url(r'^login/account-is-deactivated/?$', 'user_management.account_is_deactivated', name='user_management_account_is_deactivated'),
     url(r'^logout/?$', 'user_management.logout', name='logout'),
     url(r'^register/?$', 'user_management.register', name='register'),
+    url(r'^sign-up/?$', 'user_management.sign_up', name='sign_up'),
+    url(r'^sign-up/courts/?$', 'user_management.sign_up_courts', name='sign_up_courts'),
+    url(r'^sign-up/faculty/?$', 'user_management.sign_up_faculty', name='sign_up_faculty'),
+    url(r'^sign-up/journals/?$', 'user_management.sign_up_journals', name='sign_up_journals'),
```

Comment:
As a general note, there's a whole lot of repeated code and HTML among these four views. This invites bugs in the future, because some things have to be updated in four different places when they change (without any explicit warning), so it's easy for the views to drift. Meanwhile, other things should only be updated in one place when they change. As the code drifts (and it's not fresh in your mind, or someone else has to update it), it becomes harder to figure out whether something should be fixed in one place or four places.

Taking a random example, if I was maintaining this later, it wouldn't be clear to me why some of the templates have `$('input[type=...]:first').focus()` in the script block, while others have that commented out. It looks like that would make sense in all of them, but maybe there's a good reason for the variation that I can't see right away. If they were drawing from the same form, the template might look like `{% if some_reason %}$('input[type=text]:first').focus();{% endif %}`, making the difference between the forms self-documenting.

So, my thought for this kind of thing would be to have a single URL route, like `r'^sign-up/?(?P<group>courts|faculty|journals)?/?$'`, and then a single view like `def sign_up(request, group='default')`, and a single template, with the form logic and template varying based on the `group` variable.

Not a huge deal, just an option to keep in mind ...